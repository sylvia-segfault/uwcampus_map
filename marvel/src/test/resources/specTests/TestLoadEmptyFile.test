# Test the parser on a empty file

LoadGraph graph empty.tsv
ListNodes graph