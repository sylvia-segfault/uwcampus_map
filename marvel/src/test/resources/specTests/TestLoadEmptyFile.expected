# Test the parser on a empty file

loaded graph graph
graph contains: